src/jobs/anonymizeBeneficiaries.ts

```diff
@@ -3,63 +3,72 @@ import { Types } from 'mongoose';
 
 /** Staff resource ID */
 const AID_RESOURCE_ID = new Types.ObjectId('64e6e0933c7bf3962bf4f04c');
+const FAMILY_RESOURCE_ID = new Types.ObjectId('64de75fd3fb2a11c988dddb2');
```

Comment:
missing jsdoc, don't forget to always run npm run lint before marking the PR as ready for review